docling/datamodel/base_models.py

```diff
@@ -104,6 +106,7 @@ class OutputFormat(str, Enum):
     ],
     InputFormat.XML_USPTO: ["application/xml", "text/plain"],
     InputFormat.JSON_DOCLING: ["application/json"],
+    InputFormat.AUDIO: ["audio/wav", "audio/x-wav"],
```

Comment:
If we accept also `mp3` the mimetypes should reflect that as `audio/mpeg` and `audio/mp3`.